src/linker/Linker/Driver.cs

```diff
@@ -821,7 +821,11 @@ string GetNextStringValue ()
 		protected virtual LinkContext GetDefaultContext (Pipeline pipeline)
 		{
 			LinkContext context = new LinkContext (pipeline) {
+#if FEATURE_ILLINK
+				CoreAction = AssemblyAction.Link,
+#else
 				CoreAction = AssemblyAction.Skip,
+#endif
 				UserAction = AssemblyAction.Link,
```

Comment:
Should we change the to copy-used?

Reply:
Hmm, I think the defaults are most relevant for people who are using illink either from the command-line, or who are using the MSBuild task directly in their own targets (not through the SDK which probably defaults to copyused). I personally would want those defaults to have the "link" behavior similar to the monolinker command-line, but I can change it if you prefer. Note that most of our tests rely on the "link" behavior so I would need to tweak those as well.